examples/pcf8523/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
```

Comment:
It is best to avoid the use of `fmt` package in examples, since that requires reflection and increases the size of the resulting binary. "Plain old println()" please. :smile_cat: